PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!--
+Thanks for being here!
+Review these resources before making a contribution:
+  - Code of Conduct: https://github.com/target/markdown-inject/blob/main/CODE_OF_CONDUCT.md
+  - Contribution guide: https://github.com/target/markdown-inject/blob/main/CONTRIBUTING.md
+-->
+
+closes #<issue number>
```

Comment:
I had a comment above this explaining what it is / does, but `prettier` causes comments to be spaced off of the content by a line, moving the comment away from the `closes #` line in a way that made it not as clear what was being referred to. I chose to omit instead.

Reply:
i think its pretty clear from just the convention